galaxyui/src/app/content-detail/cards/quality-details/quality-details.component.ts

```diff
@@ -0,0 +1,108 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+class WarningType {
+    violationsCount: number;
+    message: string;
+    severityIcon: string;
+    severityText: string;
+}
+
+class Warning {
+    count: number;
+    rulesViolated: string[];
+    ruleDetails: {};
+
+    constructor() {
+        this.count = 0;
+        this.rulesViolated = [];
+        this.ruleDetails = {};
+    }
+}
+
+@Component({
+    selector: 'app-quality-details',
+    templateUrl: './quality-details.component.html',
+    styleUrls: ['./quality-details.component.less'],
+})
+export class QualityDetailsComponent implements OnInit {
+    constructor() {}
+
+    @Input()
+    content: any;
+
+    contentWarn: Warning;
+    metaWarn: Warning;
+    compatibilityWarn: Warning;
+
+    ngOnInit() {
+        this.contentWarn = new Warning();
+        this.metaWarn = new Warning();
+        this.compatibilityWarn = new Warning();
+
+        if (this.content.content_score === null) {
+            this.content.content_score = 'NA';
+        }
+
+        if (this.content.metadata_score === null) {
+            this.content.metadata_score = 'NA';
+        }
+
+        if (this.content.compatibility_score === null) {
+            this.content.compatibility_score = 'NA';
+        }
+
+        for (const el of this.content.summary_fields.task_messages) {
+            if (el.is_linter_rule_violation && el.rule_severity > 0) {
```

Comment:
My thinking on this is that we definitely want to display violations with a severity of 0 to the author, but I don't think it's necessary for the end user to know about them.

This particular view is designed to show the details behind how the score was calculated to the end user. If a severity of 0 has no effect on the score, then it's not really relevant to this end and I think it would end up being more confusing than useful. If we really want to display those warnings here, I would suggest possibly creating another section under warnings called 'info' (or something like that) for severity 0 warnings.